### PR TITLE
[TGL] Initial silicon PCH package baseline for TGL platform support

### PR DESCRIPTION
**Pull request by jinjhuli**
Build command
python BuildLoader.py build tgl

Stitch command
python Platform/TigerlakeBoardPkg/Script/StitchLoader.py
-i <EXISTING_IFWI_IMAGE>
-o <SBL_IFWI_IMAGE>
-s Outputs/tgl/SlimBootloader.bin

*Both command will only works after adding Tigerlake platform
package in upcoming patch.

Signed-off-by: jinjhuli <jin.jhu.lim@intel.com>

### CODE REVIEW

File: Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## @file
+#  Provide FSP information header
+#
+# Copyright (c) 2006 - 2016, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
```
Comment (mauricema) [inline Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf:5]: do you need FitTabl.aslc and FitTbl.inf at all?
FIT table are dynamically generated by SBL build.  It should not be required.

Comment (gdong1) [reply Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf:5]: yes. Please remove FitTbl from this package.

File: Silicon/TigerlakePchPkg/Include/PchAccess.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#define STALL_ONE_SECOND 1000000
+#endif
+#else  // SLE_FLAG
+#ifndef STALL_ONE_MICRO_SECOND
+#define STALL_ONE_MICRO_SECOND 0
+#endif
+#ifndef STALL_ONE_SECOND
+#define STALL_ONE_SECOND 0
+#endif
+#endif // SLE_FLAG
```
Comment (gdong1) [inline Silicon/TigerlakePchPkg/Include/PchAccess.h:34]: SLE_FLAG is not defined, we could remove related defination.

File: Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf (hunk 1 above)
Comment (jinjhuli) [reply Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf]: Done removed.

File: Silicon/TigerlakePchPkg/Include/Register/PchRegsFia.h [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+#define R_PCH_LP_PCR_FIA_LOS1_REG_BASE                    0x300
+#define R_PCH_LP_PCR_FIA_LOS2_REG_BASE                    0x304
+#define R_PCH_LP_PCR_FIA_LOS3_REG_BASE                    0x308
+#define R_PCH_LP_PCR_FIA_LOS4_REG_BASE                    0x30C
+#else // PCH_ICL
+#define R_PCH_LP_PCR_FIA_LOS1_REG_BASE                    0x250
+#define R_PCH_LP_PCR_FIA_LOS2_REG_BASE                    0x254
+#define R_PCH_LP_PCR_FIA_LOS3_REG_BASE                    0x258
+#define R_PCH_LP_PCR_FIA_LOS4_REG_BASE                    0x25C
+#endif // PCH_ICL
```
[the remark this answers is not in the document]
Comment (jinjhuli) [reply Silicon/TigerlakePchPkg/Include/Register/PchRegsFia.h]: Done removed.

File: Silicon/TigerlakePchPkg/Include/PchAccess.h (hunk 2 above)
Comment (jinjhuli) [reply Silicon/TigerlakePchPkg/Include/PchAccess.h]: Done removed.

**Review by gdong1 — APPROVED** [review]

**Pull request merged**